server/services/blockedDatesService.test.ts

```diff
@@ -58,4 +58,28 @@ describe('Blocked dates service', () => {
       expect(result).toStrictEqual([prisonExcludeDateDto])
     })
   })
+
+  describe('isBlockedDate', () => {
+    it('should return true for given date if exclude date found', async () => {
+      const prisonExcludeDateDto = TestData.prisonExcludeDateDto()
+      const excludedDate = prisonExcludeDateDto.excludeDate
+      orchestrationApiClient.isBlockedDate.mockResolvedValue(true)
+
+      const result = await blockedDatesService.isBlockedDate(prisonId, excludedDate, username)
+
+      expect(orchestrationApiClient.isBlockedDate).toHaveBeenCalledWith(prisonId, excludedDate)
+      expect(result).toStrictEqual(true)
+    })
```

Comment:
This works but is more complicated than it needs to be; i.e. no need to deal with `prisonExcludeDateDto`s. All that needs testing at this level is that the dependency is called with the correct parameters. So, something like this would do:

```suggestion
    it('should return boolean indicating whether given date is a blocked date', async () => {
      const date = '2000-02-01'
      orchestrationApiClient.isBlockedDate.mockResolvedValue(true)

      const result = await blockedDatesService.isBlockedDate(prisonId, date, username)

      expect(orchestrationApiClient.isBlockedDate).toHaveBeenCalledWith(prisonId, date)
      expect(result).toBe(true)
    })
```
Same for second test (for `false`) - though you could just drop this as one is probably enough because there's no different logic being tested here.